x-pack/filebeat/tests/system/test_http_endpoint.py

```diff
@@ -178,6 +179,7 @@ def test_http_endpoint_wrong_content_header(self):
         assert r.json()[
             'message'] == 'wrong Content-Type header, expecting application/json'
 
+    @unittest.skipIf(os.getenv("CI") is not None and platform.system() == 'Darwin', 'Flaky test: https://github.com/elastic/beats/issues/30028')
```

Comment:
I think that platform is in the `sys` package.
```suggestion
    @unittest.skipIf(os.getenv("CI") is not None and sys.platform.system() == 'Darwin', 'Flaky test: https://github.com/elastic/beats/issues/30028')
```

Reply:
Thanks, missed it!

Reply:
Apparently `sys.platform` is not what we're supposed to use. It's `platform` :)